lobby/src/main/java/games/strategy/engine/lobby/server/LobbyRunner.java

```diff
@@ -1,33 +1,35 @@
 package games.strategy.engine.lobby.server;
 
 import java.util.logging.Level;
-import java.util.logging.Logger;
 
 import games.strategy.engine.config.FilePropertyReader;
 import games.strategy.engine.config.lobby.LobbyPropertyReader;
 import games.strategy.sound.ClipPlayer;
+import lombok.extern.java.Log;
 
 /**
- * A 'main' class to launch the lobby server.
+ * Runs a lobby server.
  */
-public class LobbyRunner {
-  private static final Logger logger = Logger.getLogger(LobbyRunner.class.getName());
+@Log
+public final class LobbyRunner {
+  private LobbyRunner() {}
 
   /**
-   * Launches a lobby instance.
-   * Lobby stays running until the process is killed or the lobby is shutdown.
+   * Entry point for running a new lobby server. The lobby server runs until the process is killed or the lobby server
+   * is shut down via administrative command.
    */
-  public static void main(final String[] args) {
+  public static void main(final String... args) {
```

Comment:
Does this still work as entry point? 
I'm curious to know the reasoning behind this change.

Reply:
> Does this still work as entry point?

Yup, `String[]` and `String...` produce the same signature in the bytecode.

> I'm curious to know the reasoning behind this change.

No reason other than it's more modern syntax.  I thought we had an example of doing that before in the map making tools (when they could be called either in-process or out-of-process).  But since they've all been converted to in-process only, we only have three `main()` methods left in the code, and they all use `String[]`, so I'll just revert this line.

Reply:
99488ff83